docs/03-TechnicalInfrastructure.md

```diff
@@ -4,39 +4,43 @@
 ## 3.1 Back end directory structure
 
 The root directory of the imaging part of a LORIS instance is typically 
-  `/data/project`.
+  `/data/$PROJECT`.
 
 ```
 ## Imaging pipeline file directory structure
 /
 |__ data
-    |__ project
+    |__ incoming
+    |__ $PROJECT
         |__ bin
         |   |__ mri
         |__ data
-            |__ assembly
-            |__ incoming
+            |__ trashbin
+            |__ tarchive
+            |__ pic
             |__ jiv
             |__ logs
-            |__ pic
-            |__ pipelines
-            |__ protocols
-            |__ tarchive
-            |__ trashbin
+            |__ assembly
```

Comment:
The directories under the data folder should be in alphabetical order I think. Not sure why this is not the case anymore. When you do `ls` it usually lists in alphabetical order. Like the * added to the `pipelines` and `protocol` folder though!

Reply:
I changed them to be the same order as in the installer script so we do not forget any directory;
but I will revert back to alphabetical order